search-ui/src/requests/search-api.ts

```diff
@@ -110,8 +113,10 @@ export async function createDocumentAccessRequest(
 
 
 export async function getActiveAccessRequests(business_identifier: string): Promise<AccessRequestsHistoryI> {
+  const { loadAuth } = useAuth()
+  loadAuth()
```

Comment:
shouldn't this already be done via the app.vue always? This will trigger another api call to auth so would be good to take it out if possible

Reply:
could put it in the `getSearchConfig` and only when currentAccount isn't already set